notifications.md

```diff
@@ -498,7 +499,7 @@ Unlike attaching files in mailable objects, you may not attach a file directly f
      * Get the mail representation of the notification.
      *
      * @param  mixed  $notifiable
-     * @return Mailable
+     * @return \Illuminate\Notifications\Messages\MailMessage
```

Comment:
This should still be a Mailable.
Because in this Section it is about returning a Mailable instead of a MailMessage.
```suggestion
     * @return Mailable
```